schema/crdb/blueprint-add-clickhouse-tables/up1.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS omicron.public.bp_clickhouse_cluster_config (
+    blueprint_id UUID PRIMARY KEY,
+    generation INT8 NOT NULL,
+    max_used_server_id INT8 NOT NULL,
+    max_used_keeper_id INT8 NOT NULL
+    cluster_name TEXT NOT NULL
+    cluster_secret TEXT NOT NULL
```

Comment:
```suggestion
    max_used_keeper_id INT8 NOT NULL,
    cluster_name TEXT NOT NULL,
    cluster_secret TEXT NOT NULL,
```

(and the same in `dbinit.sql`)